content/en/developers/faq/data-collection-resolution-retention.md

```diff
@@ -20,6 +20,10 @@ Find below a summary of Datadog data collection, resolution, and retention:
 | Trace Events (Unsampled)     | Datadog Agent + tracing library + enabled logs                        | Real time           | 1 micro sec        | 2 Weeks                                 | APM + Logs       |
 | Logs                         | Datadog Agent + Logs, 3rd party log collectors, or API                | Real time           | 1 micro sec        | Plan                                    | Logs             |
 | API Test metrics             | Datadog Synthetics application                                        | 1 min               | 1 min              | 15 months                               | Synthetics       |
```

Comment:
Let's put `User-defined` in collection interval column for API Test metrics as well since it depends on the frequency picked by the user as well (just like API test results)